projects/L4T/devices/Switch/packages/moonlight/package.mk

```diff
@@ -0,0 +1,43 @@
+################################################################################
```

Comment:
keep original moonlight package, rename this package to e.g. `moonlight-libretro` or `lr-moonlight` or similar (to avoid duplicate package name) and adapt depends / patch names as needed

Reply:
also this package can be moved to the proper location for libretro-cores - `packages/libretro`

Reply:
Actually, this version is very switch specific, and should not be moved into the main tree. There is a whole other repo for non switch moonlight core, but this one was patched upstream for lakka switch, and old version at that. Plaidman and I just got it to build in tree, and I then applied a patch to it to force it to use nvmpi decoder over software decoder.